packages/devtools/lib/src/memory/memory.dart

```diff
@@ -68,8 +75,10 @@ class MemoryScreen extends Screen with SetStateMixin {
   MemoryChart memoryChart;
 
   CoreElement tableContainer;
+  CoreElement history; // Memory navigation history.
```

Comment:
Nit: make a dartdoc comment

Reply:
Could you also add details explaining what memory navigation history is?

Reply:
Yep, added this comment block:

  // Memory navigation history. Driven from selecting items in the list of
  // known classes, instances of a particular class and clicking on the class
  // and field that allocated the instance (holds the reference).
  // This list is displayed as a set of hyperlinks e.g.,
  //
  //     class1 (instance) > class2.extra > class3.mainHolder
  //     -----------------   ------------   -----------------
  //
  // Clicking on one of the above links would select the class and instance that
  // was associated with that hover navigation.  In this case:
  //    [class3.mainHolder] - class3 called class2 constructor storing the
  //                          reference to class2 in the field mainHolder.
  //    [class2.extra]      - class2 called class1 constructor and stored the
  //                          reference to class1 in field extra.